packages/usa-in-page-navigation/src/test/test-patterns/test-minimum-header.twig

```diff
@@ -0,0 +1,22 @@
+<div class="usa-in-page-nav-container">
+  <aside
+    class="usa-in-page-nav"
+    data-minimum-heading-count="{{ minimumHeaderCount }}"
+  ></aside>
+  <main>
+    <h2>Heading level 2</h2>
+    <p>Section content</p>
+    {% if headerLevels >= 2 %}
+    <h3>Heading level 3</h3>
+    <p>Section content</p>
+    {% endif %}
+    {% if headerLevels >= 3 %}
+    <h4>Heading level 4</h4>
+    <p>Section content</p>
+    {% endif %}
+    {% if headerLevels >= 4 %}
+    <h5>Heading level 5</h5>
+    <p>Section content</p>
+    {% endif %}
```

Comment:
**Issue (blocking):** The current tests don't work for headings above 2 levels

<img width="927" alt="image" src="https://github.com/user-attachments/assets/72273b2e-f061-49b7-a52e-c326b91964dd" />


**Suggestion** We should limit these heading levels to level 2 and 3

By default, the in-page nav selection only uses `h2` & `h3` unless unless specified otherwise using the [data-heading-elements](https://arc.net/l/quote/uivmjhkb) data-attribute.

Limiting to only `h2` and `h3` will showcase default behavior of the component a bit better! 👍

Reply:
Fixed in [ff2ce12](https://github.com/uswds/uswds/pull/6205/commits/ff2ce12aa8d92f083f068efd61641156419dec87) thanks for pointing that out :)